modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -1361,7 +1361,7 @@
         }
       }
     },
-    "/:veteranId/526": {
+    "/veterans/{veteranId}/526": {
```

Comment:
routes were different from other v2 services. you can review on dev developer portal